index.src.html

```diff
@@ -1233,6 +1233,36 @@ <h3 id="intervention-report">Intervention</h3>
       column number in [=InterventionReportBody/sourceFile=] of the offending
       behavior (which prompted the intervention), or null otherwise.
 
+  <h3 id="crash-report">Crash</h3>
+
+  <dfn>Crash reports</dfn> indicate that the user was unable to continue using
+  the page because the browser (or one of its processes necessary for the page)
+  crashed. For security reasons, no details of the crash are communicated except
+  for and a unique identifier (which can be interpreted by the browser vendor),
+  and optionally the reason for the crash (such as "oom").
+
+  <a>Crash reports</a> have the <a>report type</a> "crash".
+
+  <a>Crash reports</a> are <a>visible to <code>ReportingObserver</code>s</a>.
+
+  <pre class="idl">
+    interface CrashReportBody : ReportBody {
+      readonly attribute DOMString crashId;
+      readonly attribute DOMString? reason;
+    };
+  </pre>
+
+  An <a>crash report</a>'s [=report/body=], represented in JavaScript by
+  {{CrashReportBody}}, contains the following fields:
+
+    - <dfn for="CrashReportBody">crashId</dfn>: A unique identifier. This
+      identifier will not be meaningful to developers directly, but it can
+      potentially be supplied to the browser vendor for more details.
```

Comment:
Can you write a couple of sentences (maybe in a note paragraph) with an example?  e.g. what kind of information might the browser vendor want to encode in the crash ID.

Reply:
And do you envision there being size limitations?  Is this something where the browser vendor might encode a few KB of obfuscated JSON or something like that?  It couldn’t just be an ID, because there isn’t a way to communicate the payload that the ID refers to, right?

Reply:
Or maybe this is to hold a browser-specific error enum of some kind?

Reply:
What I am imagining is that this will be a relatively short <100 chars alphanumeric ID (I will add this to the spec). In Chrome, we will just put in our own crash ID, which we can potentially match up to a specific known crash (like in chrome://crashes).

Reply:
I've also added an example report. I will add examples for the other report types shortly.